packages/gatsby-theme-docs/src/components/search-input.js

```diff
@@ -53,7 +55,7 @@ const Input = styled.input`
 const SearchInputIcon = styled.span`
   position: absolute;
   z-index: ${designSystem.dimensions.stacks.base};
-  top: ${designSystem.dimensions.spacings.s};
+  top: ${designSystem.dimensions.specialSpacings.searchIcon};
```

Comment:
What's the need for this change?

Reply:
You need 5px to have the icon perfectly aligned with the input field. And there is no 5px value in the spacings 🤔